recipes/aws-kvs-pic/all/patches/0001-do-not-enforce-fpic.patch

```diff
@@ -0,0 +1,11 @@
+--- a/CMakeLists.txt	2021-08-06 17:10:31.808434572 -0300
++++ b/CMakeLists.txt	2021-08-06 17:10:31.888434646 -0300
+@@ -79,7 +79,7 @@
+ set(THREADS_PREFER_PTHREAD_FLAG ON)
+ find_package(Threads)
+ 
+-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
```

Comment:
The readme says the code has PIC. Should we allow the user to disable it anyways?
https://github.com/awslabs/amazon-kinesis-video-streams-pic/blob/master/README.md#note-on-alignment

Just in case, the `-pic` in the name stands for `platform independent code`. Different PIC.

Reply:
I think that section is talking about the memory allocator.
Building something PIC (or not PIC) only influences the memory addressing of loads and stores in the assembly.

I find it hard to believe if `-fPIC` would be a must-be.